mandatory/2-function-creation.js

```diff
@@ -16,7 +17,14 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  let number = parseFloat((num).toFixed(2)) + "%";
```

Comment:
Being really picky here but you may want to move line 21 so it only get executed if you reach the else block. Otherwise, it will potentially run and number won't be used if num is > 100.